meson.build

```diff
@@ -1740,6 +1740,10 @@ if get_option('libmpv')
     headers = ['libmpv/client.h', 'libmpv/render.h',
                'libmpv/render_gl.h', 'libmpv/stream_cb.h']
     install_headers(headers, subdir: 'mpv')
+
+    # Allow projects to build with libmpv by cloning into ./subprojects/mpv
+    libmpv_dep = declare_dependency(link_with: libmpv)
+    meson.override_dependency('mpv', libmpv_dep)
```

Comment:
Maybe would be better to call it `libmpv`? I think mpv is more commonly associated with standalone application.

Reply:
When using mpv in meson, the dependency is already called `mpv`, so we can't really change the name here even if we wanted to.